.github/workflows/release_prep.sh

```diff
@@ -2,6 +2,13 @@
 
 set -o errexit -o nounset -o pipefail
 
+# Don't include examples in the distribution artifact, to reduce size.
+# You may want to add additional exclusions for folders or files that users don't need.
+# NB: this mechanism relies on a `git archive` feature, which is much simpler and less
+# error-prone than using Bazel to build a release artifact from sources in the repository.
+# See https://git-scm.com/docs/git-archive#ATTRIBUTES
+echo >>.git/info/attributes "examples export-ignore"
```

Comment:
What is the motivation to write this into `.git/info/attributes` instead checking in a corresponding `.gitattributes` file?
Is it so that the GitHub autogenerated source archive still does include the examples?
Perhaps it's worth mentioning the motivation in the comment as well.